src/features/toDosSelectors.js

```diff
@@ -0,0 +1,23 @@
+import Filters from '../core/filtersEnum';
+
+export const selectToDoList = (state) => {
+  switch (state.toDoList.filter) {
+    case Filters.Active:
+      return state.toDoList.toDoList.filter((todo) => !todo.isCompleted);
+    case Filters.Completed:
+      return state.toDoList.toDoList.filter((todo) => todo.isCompleted);
+    default:
+      return state.toDoList.toDoList;
+  }
+};
+
+export const selectToDoListAmount = (state) => {
```

Comment:
You can reuse `selectToDoList` with help of https://redux-toolkit.js.org/api/createSelector